go.mod

```diff
@@ -86,9 +88,11 @@ require (
 replace (
 	github.com/googleapis/gnostic => github.com/googleapis/gnostic v0.4.1
 	github.com/metal3-io/baremetal-operator => github.com/openshift/baremetal-operator v0.0.0-20210409032903-31b989a197eb // Use OpenShift fork
+	github.com/openshift/assisted-service/models => ./models
 	go.etcd.io/bbolt => go.etcd.io/bbolt v1.3.5
 	go.etcd.io/etcd => go.etcd.io/etcd v0.5.0-alpha.5.0.20200910180754-dd1b699fc489 // ae9734ed278b is the SHA for git tag v3.4.13
 	google.golang.org/grpc => google.golang.org/grpc v1.27.1
+	gorm.io/gorm => gorm.io/gorm v1.22.3
```

Comment:
isn't it inconsistent with the models' version?

Reply:
yes but in the models it have a new version that caused me some issues with unit tests, i wrote in the comment 
```
require specific gorm version because i had some issue with timezone
translation
```

Reply:
Ah, I thought you meant to use a specific gorm version, but the same version on both modules

Reply:
The gorm version in under model is newer `gorm.io/gorm v1.23.1` 
because the rood model import from it so it takes the same version, but it cause some issue in the code, so i enforce specific version in the root mod file.